config.ru

```diff
@@ -1,5 +1,10 @@
 require 'mumukit'
 
+module Mumukit::WithCommandLine
+  def limit_command
+    '.heroku/vendor/bin/limit'
```

Comment:
I actually wanted to include to limit script as part of the mumukit gem, but I could make it work.